tests/test_get_namespace.py

```diff
@@ -12,3 +12,10 @@ def test_get_namespace(library):
     expected_namespace = getattr(array_api_compat, library)
     assert namespace is expected_namespace
 
+
+@pytest.mark.parametrize("array_namespace", ["cupy.array_api", "numpy.array_api"])
+def test_get_namespace_returns_actual_namespace(array_namespace):
```

Comment:
Why is this a separate test from the one above?

Reply:
Going to just merge this and fix this and the other issues you opened issues for in another PR.